mars/services/task/execution/ray/executor.py

```diff
@@ -236,6 +258,8 @@ async def execute_subtask_graph(
             if not isinstance(chunk.op, Fetch)
         }
         for subtask in subtask_graph.topological_iter():
+            if self._cancelled is True:  # pragma: no cover
+                raise asyncio.CancelledError()
```

Comment:
We can get current task by `asyncio.current_task` at the beginning of `execute_subtask_graph`. Then, `cancel()` just cancel current execute task.

Reply:
Be aware that the `self._cancelled` should be checked at the beginning of `execute_subtask_graph`, too.

Reply:
Good suggestion, i'll fix it.